spec/models/progress_message_spec.rb

```diff
@@ -0,0 +1,27 @@
+describe ProgressMessage, :type => :model do
+  let(:tenant) { create(:tenant) }
+  let(:order_item1) { create(:order_item, :tenant_id => tenant.id, :portfolio_item_id => 1, :order_id => 1, :owner => "jdoe") }
```

Comment:
@lindgrenj6 I don't think you need to set the owner, the default will be set properly from the mixing